django bootstrap is unable to render invalid form

When a form with invalid `vote_value` is passed to `proposal_reviewer_vote` form, bootstrap thorws up with following traceback

```python
>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>> traceback >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>

self = <tests.integrations.test_reviewer_views.TestReviewerViews instance at 0x112941fc8>, settings = <pytest_django.fixtures.MonkeyPatchWrapper object at 0x112332cd0>
login = (<tests.fixtures._Client object at 0x112443790>, <User: username>), conferences = {'future': <Conference: Future>, 'past': <Conference: Past>}, create_proposal = <Proposal: , Workshop>, create_reviewer = <User: username>

    def test_post_review_proposal_vote_with_invalid_data(self, settings, login,
                                                         conferences,
                                                         create_proposal,
                                                         create_reviewer):
        client = login[0]
        conference = conferences['future']

        proposal = create_proposal

        kwargs = {'conference_slug': conference.slug,
                  'proposal_slug': proposal.slug}
        url = reverse('proposal-reviewer-vote', kwargs=kwargs)
        data = {'vote_value': 12}
>       response = client.post(url, data)

tests/integrations/test_reviewer_views.py:206:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
../../.virtualenvs/junction/lib/python2.7/site-packages/django/test/client.py:482: in post
    secure=secure, **extra)
../../.virtualenvs/junction/lib/python2.7/site-packages/django/test/client.py:296: in post
    secure=secure, **extra)
../../.virtualenvs/junction/lib/python2.7/site-packages/django/test/client.py:358: in generic
    return self.request(**r)
../../.virtualenvs/junction/lib/python2.7/site-packages/django/test/client.py:440: in request
    six.reraise(*exc_info)
../../.virtualenvs/junction/lib/python2.7/site-packages/django/core/handlers/base.py:111: in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
../../.virtualenvs/junction/lib/python2.7/site-packages/django/contrib/auth/decorators.py:21: in _wrapped_view
    return view_func(request, *args, **kwargs)
../../.virtualenvs/junction/lib/python2.7/site-packages/django/views/decorators/http.py:41: in inner
    return func(request, *args, **kwargs)
junction/proposals/votes_views.py:148: in proposal_reviewer_vote
    'form_errors': form.errors})
../../.virtualenvs/junction/lib/python2.7/site-packages/django/shortcuts.py:50: in render
    return HttpResponse(loader.render_to_string(*args, **kwargs),
../../.virtualenvs/junction/lib/python2.7/site-packages/django/template/loader.py:178: in render_to_string
    return t.render(context_instance)
../../.virtualenvs/junction/lib/python2.7/site-packages/django/template/base.py:148: in render
    return self._render(context)
../../.virtualenvs/junction/lib/python2.7/site-packages/django/test/utils.py:88: in instrumented_test_render
    return self.nodelist.render(context)
../../.virtualenvs/junction/lib/python2.7/site-packages/django/template/base.py:844: in render
    bit = self.render_node(node, context)
../../.virtualenvs/junction/lib/python2.7/site-packages/django/template/debug.py:80: in render_node
    return node.render(context)
../../.virtualenvs/junction/lib/python2.7/site-packages/django/template/loader_tags.py:126: in render
    return compiled_parent._render(context)
../../.virtualenvs/junction/lib/python2.7/site-packages/django/test/utils.py:88: in instrumented_test_render
    return self.nodelist.render(context)
../../.virtualenvs/junction/lib/python2.7/site-packages/django/template/base.py:844: in render
    bit = self.render_node(node, context)
../../.virtualenvs/junction/lib/python2.7/site-packages/django/template/debug.py:80: in render_node
    return node.render(context)
../../.virtualenvs/junction/lib/python2.7/site-packages/django/template/loader_tags.py:65: in render
    result = block.nodelist.render(context)
../../.virtualenvs/junction/lib/python2.7/site-packages/django/template/base.py:844: in render
    bit = self.render_node(node, context)
../../.virtualenvs/junction/lib/python2.7/site-packages/django/template/debug.py:80: in render_node
    return node.render(context)
../../.virtualenvs/junction/lib/python2.7/site-packages/django/template/base.py:1136: in render
    return func(*resolved_args, **resolved_kwargs)
../../.virtualenvs/junction/lib/python2.7/site-packages/bootstrap3/templatetags/bootstrap3.py:281: in bootstrap_form
    return render_form(*args, **kwargs)
../../.virtualenvs/junction/lib/python2.7/site-packages/bootstrap3/forms.py:45: in render_form
    return renderer_cls(form, **kwargs).render()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <bootstrap3.renderers.FormRenderer object at 0x1127dc150>, form = '', args = (), kwargs = {}

    def __init__(self, form, *args, **kwargs):
        if not isinstance(form, BaseForm):
            raise BootstrapError(
>               'Parameter "form" should contain a valid Django Form.')
E           BootstrapError: Parameter "form" should contain a valid Django Form.

../../.virtualenvs/junction/lib/python2.7/site-packages/bootstrap3/renderers.py:134: BootstrapError
>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>> entering PDB >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
> /Users/krace/.virtualenvs/junction/lib/python2.7/site-packages/bootstrap3/renderers.py(134)__init__()
```

@ChillarAnand can you have a look ?

--- a/junction/proposals/votes_views.py
+++ b/junction/proposals/votes_views.py
@@ -144,7 +144,7 @@ def proposal_reviewer_vote(request, conference_slug, proposal_slug):
                 conference=conference)
         ctx = {
             'proposal': proposal,
-            'proposal_vote_form': proposal_vote_form,
+            'form': proposal_vote_form,
             'vote': vote,
         }
 
@@ -153,10 +153,10 @@ def proposal_reviewer_vote(request, conference_slug, proposal_slug):
     # POST Workflow
     form = ProposalReviewerVoteForm(data=request.POST, conference=conference)
     if not form.is_valid():
-        return render(request, 'proposals/vote.html',
-                      {'form': form,
-                       'proposal': proposal,
-                       'form_errors': form.errors})
+        ctx = {'form': form,
+               'proposal': proposal,
+               'form_errors': form.errors}
+        return render(request, 'proposals/vote.html', ctx)
 
     # Valid Form
     vote_value = form.cleaned_data['vote_value']

--- a/junction/templates/proposals/vote.html
+++ b/junction/templates/proposals/vote.html
@@ -40,7 +40,7 @@
                     <h4 class="fill-blue text-center">Proposal Voting</h4>
                     <hr class="hr-mini">
                     <form method="POST" action="."> {% csrf_token %}
-                        {% bootstrap_form proposal_vote_form %}
+                        {% bootstrap_form form %}
                         {% buttons %}
                         <div class="text-center">
                             <button type="submit" class="btn btn-primary">
@@ -55,4 +55,3 @@
         </div>
     </div>
 {% endblock %}
-

--- a/tests/integrations/test_reviewer_views.py
+++ b/tests/integrations/test_reviewer_views.py
@@ -187,26 +187,26 @@ class TestReviewerViews:
         context = response.context
 
         assert response.status_code == 200
-        assert context['proposal_vote_form'].initial['vote_value'] == vote_value
-        assert context['proposal_vote_form'].initial['comment'] == comment
+        assert context['form'].initial['vote_value'] == vote_value
+        assert context['form'].initial['comment'] == comment
 
-    # def test_post_review_proposal_vote_with_invalid_data(self, settings, login,
-    #                                                      conferences,
-    #                                                      create_proposal,
-    #                                                      create_reviewer):
-    #     client = login[0]
-    #     conference = conferences['future']
+    def test_post_review_proposal_vote_with_invalid_data(self, settings, login,
+                                                         conferences,
+                                                         create_proposal,
+                                                         create_reviewer):
+        client = login[0]
+        conference = conferences['future']
 
-    #     proposal = create_proposal
+        proposal = create_proposal
 
-    #     kwargs = {'conference_slug': conference.slug,
-    #               'proposal_slug': proposal.slug}
-    #     url = reverse('proposal-reviewer-vote', kwargs=kwargs)
-    #     data = {'vote_value': 12}
-    #     response = client.post(url, data)
+        kwargs = {'conference_slug': conference.slug,
+                  'proposal_slug': proposal.slug}
+        url = reverse('proposal-reviewer-vote', kwargs=kwargs)
+        data = {'vote_value': 12}
+        response = client.post(url, data)
 
-    #     assert response.status_code == 200
-    #     assert 'vote_value' in response.context['form_errors']
+        assert response.status_code == 200
+        assert 'vote_value' in response.context['form_errors']
 
 
 def test_public_comment(settings, login, conferences,

